ble_controller/include/ble_controller_hci_vs.h

```diff
@@ -312,6 +321,23 @@ uint8_t hci_vs_cmd_zephyr_read_version_info(hci_vs_cmd_zephyr_read_version_info_
  */
 uint8_t hci_vs_cmd_zephyr_read_supported_commands(hci_vs_cmd_zephyr_read_supported_commands_return_t * p_return);
 
+/** @brief Zephyr Write BD ADDR.
+ *
+ * This command writes the BD_ADDR (Bluetooth public device address) value
+ * to the volatile memory. It is kept over Reset and can be read out by
```

Comment:
It is kept over an HCI Reset Command, but not a System Reset. This should be clarified.